src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml

```diff
@@ -4,15 +4,23 @@
 
     @if (!Model.HomeTeamId.HasValue && !Model.AwayTeamId.HasValue)
     {
-        <div class="row">
-            <div class="col-md-10">
-                <strong>  @Resources.UI.TournamentViews.HomeTeamPlaceholder </strong>
-            </div>
-        </div>
-
-        <div class="row">
-            <div class="col-md-10">
-                <strong>  @Resources.UI.TournamentViews.AwayTeamPlaceholder </strong>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-3 game-number">
+                    <div>
```

Comment:
what the reason of putting inside empty div here and in all other places? if you need some specific styling (like display:block or display: inline-block solve this with styling not markup)